spec/models/rent_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+describe Rent do
+  subject(:rent) { create(:rent) }
+
+  it { is_expected.to validate_presence_of :user_id }
+  it { is_expected.to validate_presence_of :book_id }
```

Comment:
same here with what I said in the model validation
Also, parenthesis are sad that you forgot about them haha

Reply:
Sorry, I have the habit of don't use parentheses.